### PR TITLE
Sanity suites for live cdn testing

### PR DESCRIPTION
**Pull request by sunilangadi2**
# Description
It is the new changes from PR #316
polarion link: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573686
Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [x] Create a test case in Polarion reviewed and approved.
- [x] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [x] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarin Test with Automation script details and update automation fields
- [x] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

**Comment by sunilangadi2** [comment]
Thanks @VasishtaShastry for this PR #339
Currently we have some issue with cdn credentials subscription to VM's 
logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1622792240101/install_ceph_pre-requisites_0.log 
tried `is_production: True` it's getting failed for RestlibException: HTTP error (404 - Not Found): Pool with id 8a99f9af795d57ab01797e572e860569 could not be found.

Hence tested these suites with `is_production: False` 
logs: 
sanity_container http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1623217299993
sanity_rpm http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1622801610473

**Comment by psathyan** [comment]
@sunilangadi2 any update on the getting a valid pool id? This PR has been pending for quite some time now. Let me know how we can be of help.

**Comment by rakeshgm** [comment]
we need to mention a valid pool id, you can start to make the changes in `install_prereq.py`.  take it from cdn creds.

**Comment by sunilangadi2** [comment]
@rakeshgm Can you please let me know what is the valid pool id we need to use?
also not sure what changes to make in **install_prereq.py** for cdn creds with `is_production: True`
Can you please help me out?

**Comment by sunilkumarn417** [comment]
> @rakeshgm Can you please let me know what is the valid pool id we need to use?
> also not sure what changes to make in **install_prereq.py** for cdn creds with `is_production: True`
> Can you please help me out?

Hi @sunilangadi2 , 

We had received information about stage and CDN settings from `Ken`. can you please take a look at that?

cc: @VasishtaShastry

**Comment by sunilangadi2** [comment]
Success logs with `is_production: True` in suite file 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1626082628774

**Comment by sunilangadi2** [comment]
@rakeshgm and @psathyan Please review the changes.

File: tests/misc_env/install_prereq.py [hunk 1]
```diff
@@ -147,17 +147,17 @@ def setup_subscription_manager(ceph, is_production=False, timeout=1800):
             command = "sudo subscription-manager --force register "
             if is_production:
                 command += "--serverurl=subscription.rhsm.redhat.com:443/subscription "
-
                 username_ = config_["cdn_credentials"]["username"]
                 password_ = config_["cdn_credentials"]["password"]
+                pool_id = "8a85f98960dbf6510160df23eb447470"
```
Comment (sunilkumarn417) [inline tests/misc_env/install_prereq.py:152]: can we pass pool Id from location where we fetch creds?

File: tests/misc_env/install_prereq.py [hunk 2]
```diff
@@ -147,17 +147,17 @@ def setup_subscription_manager(ceph, is_production=False, timeout=1800):
...
 
             else:
                 command += (
                     "--serverurl=subscription.rhsm.stage.redhat.com:443/subscription "
                 )
-
                 username_ = config_["stage_credentials"]["username"]
                 password_ = config_["stage_credentials"]["password"]
+                pool_id = "8a99f9af795d57ab01797e572e860569"
```
Comment (sunilkumarn417) [inline tests/misc_env/install_prereq.py:160]: same as above

File: suites/nautilus/live/sanity_container.yaml [hunk 3]
```diff
@@ -0,0 +1,106 @@
+tests:
```
Comment (sunilkumarn417) [inline suites/nautilus/live/sanity_container.yaml:1]: Can we list out out steps in test suite as defined in the this suite, https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier1_cephadm_upgrade.yaml#L1-L28

Comment (sunilkumarn417) [reply suites/nautilus/live/sanity_container.yaml:1]: please add steps w.r.t setting up environment for ceph live testing.

File: tests/misc_env/install_prereq.py (hunk 1 above)
Comment (rakeshgm) [reply tests/misc_env/install_prereq.py:152]: i would suggest the same, but there are places where the pool ids are hardcoded in the code. we can plan to unify all the pool ids into a file and take it from there. I think that can be separate PR. 
but nonetheless.

File: suites/nautilus/live/sanity_container.yaml (hunk 3 above)
Comment (sunilangadi2) [reply suites/nautilus/live/sanity_container.yaml:1]: @sunilkumarn417 Thanks for the hint, sure will add those steps in the suite

Comment (sunilangadi2) [reply suites/nautilus/live/sanity_container.yaml:31]: @sunilkumarn417 It's done, Please review the changes.

**Review by psathyan — APPROVED** [review]
Looks good to me

**Review by sunilkumarn417 — APPROVED** [review]

**Pull request merged**